tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -48,16 +48,13 @@ sub run() {
     };
 
     # join openqa channel
-    if (sle_version_at_least('12-SP2')) {
+    send_key "ctrl-j";
+    if (!check_screen 'empathy-join-room', 4) {
```

Comment:
FYI,
>  1     # quit
>  2     wait_screen_change { send_key "ctrl-q" };
>  3     assert_screen [qw(empathy-menu generic-desktop)];
>  4     if (match_has_tag 'empathy-menu') {
>  5         assert_and_click 'empathy-menu';
>  6         assert_and_click 'empathy-menu-quit';
>  7     }
didn't work.. http://rwmanosvm1.suse.cz/tests/223#step/empathy_irc/31
I'll reintroduce wait_still_screen 2 instead of wait_screen_change..